### PR TITLE
Use ClusterWizardToolbar in ClusterDetails

### PR DESCRIPTION
**Pull request by jtomasek**
- make 'cluster' prop optional in ClusterWizardToolbar
- Fix SNO checkbox in NewClusterPage and ClusterPage components

### CODE REVIEW

File: src/components/clusterWizard/ClusterDetails.tsx [hunk 1]
```diff
@@ -274,6 +249,18 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
                       isRequired
                     />
                   )}
+                  {isSingleNodeOpenshiftEnabled && (
```
Comment (mareklibra) [inline src/components/clusterWizard/ClusterDetails.tsx:252]: Let's merge this after release and rebase on top of #460

File: src/components/clusters/ClusterPage.tsx [hunk 2]
```diff
@@ -184,17 +187,19 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
...
-          isOpen={cancelInstallationModalOpen}
-          onClose={() => setCancelInstallationModalOpen(false)}
-          clusterId={cluster.id}
-        />
-        <ResetClusterModal
-          isOpen={resetClusterModalOpen}
-          onClose={() => setResetClusterModalOpen(false)}
-          cluster={cluster}
-        />
+        <FeatureGateContextProvider features={features}>
```
Comment (mareklibra) [inline src/components/clusters/ClusterPage.tsx:190]: Let's rebase when #459 lands.

File: src/components/clusterWizard/ClusterDetails.tsx (hunk 1 above)
Comment (jtomasek) [reply src/components/clusterWizard/ClusterDetails.tsx]: resolved

File: src/components/clusters/ClusterPage.tsx (hunk 2 above)
Comment (jtomasek) [reply src/components/clusters/ClusterPage.tsx]: resolved

**Review by mareklibra — APPROVED** [review]

**Pull request merged**